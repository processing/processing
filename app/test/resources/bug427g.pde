static final boolean DEBUG = true;

void setup() {
  MyClass x = new MyClass();
}

public class MyClass {
    public MyClass() {
        if (DEBUG) println("Debug");
    }
}


