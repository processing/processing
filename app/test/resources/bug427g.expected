import processing.core.*; 
import processing.xml.*; 

import java.applet.*; 
import java.awt.Dimension; 
import java.awt.Frame; 
import java.awt.event.MouseEvent; 
import java.awt.event.KeyEvent; 
import java.awt.event.FocusEvent; 
import java.awt.Image; 
import java.io.*; 
import java.net.*; 
import java.text.*; 
import java.util.*; 
import java.util.zip.*; 
import java.util.regex.*; 

public class bug427g extends PApplet {

static final boolean DEBUG = true;

public void setup() {
  MyClass x = new MyClass();
}

public class MyClass {
    public MyClass() {
        if (DEBUG) println("Debug");
    }
}


    static public void main(String args[]) {
        PApplet.main(new String[] { "--bgcolor=null", "bug427g" });
    }
}
